healthcheck/client_test.go

```diff
@@ -0,0 +1,59 @@
+package healthcheck
+
+import (
+	"errors"
+	"net/http"
+	"testing"
+
+	"github.com/ONSdigital/go-ns/healthcheck/mock_healthcheck"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestUnitHealthcheckClient(t *testing.T) {
```

Comment:
Seems a bit odd to have 'Unit' in the name here. I see its used elsewhere in the package though. If using golang IDE you can press `CTRL + space` after the keyword `func` and it will suggest test names in the typical convention.